src/nomad_material_processing/vapor_deposition/__init__.py

```diff
@@ -62,6 +64,164 @@
 m_package = Package(name='Vapor Deposition')
 
 
+class InsertReduction(ArchiveSection):
+    """
+    The reduction that sometimes is used to lodge the substrate in the substrate holder position..
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    insert_id = Quantity(
+        type=str,
+        description="""
+        The ID of the insert that is placed in this position to accomodate the substrate.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    material = Quantity(
+        type=str,
+        description="""
+        The material of the insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    inner_geometry = SubSection(
+        section_def=Geometry,
+    )
+    outer_geometry = SubSection(
+        section_def=Geometry,
+    )
+
+class SubstrateHolderPosition(ArchiveSection):
+    """
+    One casing position of the substrate holder.
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this position.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    slot_geometry = SubSection(
+        section_def=Geometry,
+    )
+    insert_reduction = SubSection(
+        section_def=InsertReduction,
+    )
+
+
+class SubstrateHolder(ArchiveSection):
+    """
+    The holder for the substrate.
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this position.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    lab_id = Quantity(
+        type=str,
+        description="""
+        The lab ID of the substrate holder.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    material = Quantity(
+        type=str,
+        description="""
+        The material of the holder.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    thickness = Quantity(
```

Comment:
Is this the thickness to the back of the substrates? In that case we should probably specify this.

Reply:
I will